# Metagenomics-Workshop-Spring-2020
NCGAS is hosting a metagenomics workshop, focusing on whole genome sequencing metagenome analysis between April 20th to 22nd at Indiana University, Bloominton. 

**Dates** – April 20th to 22nd, 2020 \
**Location** – Bloomington, IN \
**Registration** – https://redcap.uits.iu.edu/surveys/?s=EXAHRFYCML \
**Deadline to apply** - February 25th, 2020 

# Workshop Information 
The workshop will include discussions, lectures and hands-on tutorials, to cover topics important to constructing and analyzing shotgun sequenced metagenomic datasets. Material covers availability of HPC resources, how to access these resources, while learning how to assemble and annotate metagenomics datasets. The goal of this workshop is to provide a comprehensive preparation for metagenomic analysis and other bioinformatic tasks.
This workshop will include the basic steps in metagenomic analysis: to assemble and annotate datasets to identify taxa and function. While the material will make heavy use of XSEDE and IU machines, the material is transferable to any cluster.
Participation in this workshop is by invitation—you must register to be considered. Participation is limited by facility space.

# Prerequisites
This workshop is geared for beginners, but basic Unix commands will not be covered. As such, participants must have basic Linux functionality (sign in, moving around file system, etc.), but expertise is not required. It would be helpful if participants had some exposure to using a cluster for compute jobs and an idea of the end goals for their data.

If you are working on a Windows machine, make sure to have Putty (https://www.putty.org/), or have Bash Desktop app. 

Beginner tutorials in Linux (bash) can be found here: \
Getting Started on Unix Walkthrough (pdf) (http://pages.iu.edu/~ss93/tutorials/GettingStartedonUnixWalkthrough.pdf) \
Code Academy (https://www.codecademy.com/catalog/subject/all) 

# Objectives
Participants should leave with the following knowledge:
- Familiarity with nationally available compute resources
- An understanding of the differences, pros, and cons of VMs, Gateways, Clusters, and Clouds
- How to run and optimize a job submission on a cluster 
- How to manage large data sets and move data between resources 
- Quality control of data
- How to run metagenomics assembly
- Ability to reconstruct metagenome assembled genomes and check their quality 
- Taxa and functional annoatation of metagenomes 
- How to get help for both genomic and computational questions 

Participant data will not be analyzed during the workshop (that takes more time than we have!), but the entire pipeline will be used by participants with smaller scale demo data. We can also schedule online meetings after the workshop to discuss your data. 

# Schedule 
Coming up! 

# Presentations 
Coming up!

# Contact us 
If you have any questions, email us at help@ncgas.org.  \
Website https://ncgas.org/
\
Twitter - @ncgas \
Facebook - @ncgasiu
